core/storage/audit/gae_models.py

```diff
@@ -60,7 +60,7 @@ def get_model_association_to_user():
 
     @classmethod
     def get_export_policy(cls):
-        """Model does not contain user data."""
+        """Model doesn't contain any data directly corresponding to a user."""
```

Comment:
Wrong comment, it contains data about the query for the user?

Reply:
I'm not familiar with this model. Do we need to write an export_data for this? Or does the comment just need to be updated such that it indicates why we do not export it?

Reply:
@vojtechjelinek I think this is the only thing pending on this PR. Once we clear this up, I think this is good to merge.

Reply:
You can check my comment here https://github.com/vojtechjelinek/oppia/blob/wipeout-audit-fixes3/core/storage/audit/gae_models.py#L52-L56